src/SFA.DAS.Campaign.UnitTests/Web/Controllers/EmployerInform/HowTheyWork/WhenHowTheyWorkGetCalled.cs

```diff
@@ -0,0 +1,45 @@
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Campaign.Web.Controllers.EmployerInform;
+using SFA.DAS.Campaign.Web.Helpers;
+
+namespace SFA.DAS.Campaign.UnitTests.Web.Controllers.EmployerInform.HowTheyWork
+{
+    [TestFixture]
+    public class WhenHowTheyWorkGetCalled
+    {
+        [Test]
+        public void ThenViewIsReturned()
+        {
+            var sessionService = new Mock<ISessionService>();
+            sessionService.Setup(ss => ss.Get<LevyOptionViewModel>("LevyOptionViewModel")).Returns(new LevyOptionViewModel
+            {
+                LevyStatus = LevyStatus.Levy
+            });
+            
+            var controller = new HowDoTheyWorkController(sessionService.Object);
+
+            var result = controller.Index();
+
+            result.Should().BeOfType<ViewResult>();
+            result.As<ViewResult>().Model.Should().BeOfType<LevyOptionViewModel>();
+            result.As<ViewResult>().Model.As<LevyOptionViewModel>().LevyStatus.Should().Be(LevyStatus.Levy);
```

Comment:
Should this be a separate test?

Reply:
I've removed the `result.Should().BeOfType<ViewResult>();` as that is implicit in the following asserts.  Both of those are required to assert that the VM returned in the view is correct.  I've also renamed the tests to make the intent clearer.